docs/upcoming_changes/9074.new_feature.rst

```diff
@@ -0,0 +1,5 @@
+
+Add np.trim_zeros
+=================
+
+Pull request 9074 adds ``np.trim_zeros()`` support to Numba.
```

Comment:
I think there was a recent discussion on the format of the towncrier snippets where the conclusion was along the lines that we should keep it brief by avoiding self-references - e.g.:

```suggestion
Support for ``np.trim_zeros()`` is added.
```